vendor/src/github.com/docker/libnetwork/endpoint_info.go

```diff
@@ -246,6 +266,15 @@ func (epi *endpointInterface) Address() *net.IPNet {
 	return types.GetIPNetCopy(epi.addr)
 }
 
+func (i *endpointInterface) ExtraAddresses() []*net.IPNet {
```

Comment:
`ExtraAddresses()` uses `i.Lock()` and `defer i.Unlock()`, but this method doesn't - is that intentional?